app/Jobs/Payment/EmailPayment.php

```diff
@@ -35,12 +35,12 @@ class EmailPayment implements ShouldQueue
      *
      * @return void
      */
-    public function __construct(Payment $payment, BuildEmail $emailBuilder, $contact)
-    {
+     public function __construct(Payment $payment, $email_builder, $contact)
+     {
         $this->payment = $payment;
+        $this->emailBuilder = $email_builder;
```

Comment:
we are using snake_case for variable names, this would look like $this->email_builder;